src/packages/cli/tests/args.test.ts

```diff
@@ -0,0 +1,56 @@
+import assert from "assert";
+import args from "../src/args";
+
+describe("args", () => {
```

Comment:
YAY! CLI TESTS!

You should remove the `test/index.test.ts` file now, as it was a placeholder for tests.

Since these tests only test that the `detach` option is set, and not if it works, let's find a decent `describe` title or `it` titles that makes that clear.